File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.java
class Solution {
    public int removeDuplicates(int[] nums) {
        int i,j;
        int n=nums.length;
        for(i=0;i<n-1;i++){
            if(nums[i]==nums[i+1]){
                for(j=i;j<n-1;j++){
                    nums[j]=nums[j+1];
                }
                n--;
                i--;
            }
        }
        return n;
    }
}

File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        int n=nums.size();
        vector<int> stack;
        int i;
        for(i=0;i<n-1;i++){
            if(nums[i+1]!=nums[i]){
                stack.emplace_back(nums[i]);
            }
        }
        stack.emplace_back(nums[i]);
        int m=stack.size();
        for(int i=0;i<m;i++){
            nums[i]=stack[i];
        }
        return m;
    }
};